internal/pkg/api/server/templates/dependencies/dependencies_test.go

```diff
@@ -175,8 +181,11 @@ func TestForProjectRequest_TemplateRepository_Cached(t *testing.T) {
 	mockedDeps.DebugLogger().Truncate()
 
 	// Old FS is deleted (nobody uses it)
-	time.Sleep(200 * time.Millisecond)
-	assert.NoDirExists(t, repo3.Fs().BasePath())
+	assert.Eventually(t, func() bool {
+		// NoDirExists
+		_, err := os.Stat(repo3.Fs().BasePath()) // nolint: forbidigo
+		return err == os.ErrNotExist
+	}, 10*time.Second, 100*time.Millisecond)
```

Comment:
Tu bol problem, ze na Windows sa ten adresar za `200ms` nezmazal.
Tak sa to kontroluje kazdych `100ms` a maximalne `5s`.
Mazanie bezi v goroutine na pozadi, ked sa uvolni zamok - nikto dany adresar uz nepouziva.
Netrapi nas, aby to bolo zmazane hned, ale aby to bolo zmazane a nebralo to miesto na disku.

Reply:
(uvidime ci to tak bude stacit)

Reply:
:heavy_check_mark: